### ISSUE TITLE
ToggleMenuItem violates a11y rules

### ISSUE THREAD

**Issue by t3chguy** [element-hq/compound#290]
<img width="1108" alt="image" src="https://github.com/element-hq/compound/assets/2403652/adc19cbf-3895-462b-8c34-cbebc48fa33b">
<img width="1238" alt="image" src="https://github.com/element-hq/compound/assets/2403652/6aa6b666-b753-4003-8af4-c18cdc5c5b16">

**Comment by daniellekirkwood** [comment]
@t3chguy this is also the right panel, right?

**Comment by t3chguy** [comment]
For Element Web today yes, but it affects all consumers of Compound which use this component
